Infrastructure/terraform.tfstate

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
can you remove this file. It should be ignored by git. 

Maybe just remove it and it should work

Reply:
Removed the terraform.tfstate file